modules/standard/main.tf

```diff
@@ -87,17 +87,29 @@ resource "aws_iam_policy" "s3" {
   policy = local.s3_policy_arn
 }
 
+resource "aws_iam_policy" "vpc" {
+  count  = var.vpc_security_group_ids != null && local.create_role ? 1 : 0
+  name   = "${var.function_name}-vpc-policy"
+  policy = data.aws_iam_policy_document.vpc[0].json
+}
+
 resource "aws_iam_role_policy_attachment" "logs" {
-  count = local.create_role ? 1 : 0 #local.cloudwatch_policy_arn != null ? 1 : 0
+  count      = local.create_role ? 1 : 0 #local.cloudwatch_policy_arn != null ? 1 : 0
   #name       = "${local.role_name}-logs"
   role       = aws_iam_role.lambda[0].name
   policy_arn = aws_iam_policy.logs[0].arn
 
 }
 
 resource "aws_iam_role_policy_attachment" "s3" {
-  count = local.s3_policy_arn != null ? 1 : 0
+  count      = local.s3_policy_arn != null ? 1 : 0
   #name       = "${local.role_name}-s3"
   role       = aws_iam_role.lambda[0].name
   policy_arn = aws_iam_policy.s3[0].arn
 }
+
+resource "aws_iam_role_policy_attachment" "vpc" {
+  count      = var.vpc_security_group_ids != null && local.create_role ? 1 : 0
```

Comment:
maybe move `var.vpc_security_group_ids != null && local.create_role` to some local variable

Reply:
I would put whole "var.vpc_subnet_ids != null && var.vpc_security_group_ids != null" condition (used in main.tf) to some local.is_in_vpc to better visibility. Then can be also some combination of local.is_in_vpc with local.create_role into common bool local

Reply:
I plan to remove variable create_function completely and will do refactor then. I've checked usage of this repo and nobody is using it and it adds only complexity here.